datasimulator/main.py

```diff
@@ -144,9 +143,13 @@ def run_submission_order_generation(graph, data_path, node_name=None):
     else:
         submission_order = graph.generate_submission_order()
 
-    with open(os.path.join(data_path, "DataImportOrderPath.txt"), "w") as outfile:
-        for node in submission_order:
-            outfile.write(node.name + "\t" + node.category + "\n")
+    file_path = os.path.join(data_path, "DataImportOrderPath.txt")
+    path_exists = os.path.exists(data_path)
+    if not path_exists:
+        logger.error("Cannot create file because path does not exist. Did you create a 'test-data' folder?")
```

Comment:
i think a better solution in terms of UX would be to create the directory automatically instead

Reply:
I suggested that as well but @k-burt-uch is of the mind that we should just handle the error a bit more gracefully.